backend/main/server/controllers.go

```diff
@@ -106,12 +106,8 @@ func (a *App) getVotesForAddress(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	start, count := getPageParams(r.FormValue("start"), r.FormValue("count"), 25)
-	orderParams := shared.OrderedPageParams{
-		Start: start,
-		Count: count,
-		Order: "desc",
-	}
+	orderParams := getPageParams(r.FormValue("start"), r.FormValue("count"), 25)
```

Comment:
Could we reuse getOrderdPageParams and just pass in "desc" as the parameter? Could we get merge `getPageParams` and `getOrderedPageParams` and pass in a default when we don't have order passed through the API?